docs/Misc/GameBuildDifferences.md

```diff
@@ -33,3 +33,35 @@ While not always the case with games, there are a handful of differences between
 
 For example, the versions build on Oct 16th and later have the powerful Chaos Control Glitch patched. Making the US GameCube and PS2 versions the only ones that can perform that glitch.
 
+The Oct 16th and later versions also have a few minor level object (layout) differences, extra objects are added as documented below.
+
+### Oct 16th Layout Changes
+
+|Layout File|Stage & Type|Change
+|--|--|--|
+stg0201/stg0201_cmn.dat|Glyphic Canyon (All)|Extra Killplane Collision (6) on Secret KeyDoor route|
+stg0300/stg0300_hrd.dat|Cryptic Castle (Expert)|ChaosControlStop Trigger at Chao Room / Dark route|
+stg0401/stg0401_cmn.dat|The Doom (All)|Bugfix/GUN Robot (170) sets RenderDist 7 -> 5 to prevent it falling through floor, *possibly also "Unknown" as well|
+stg0403/stg0403_nrm.dat|Mad Matrix (Normal)|Unknown|
+stg0501/stg0501_cmn.dat|Air Fleet (All)|Unknown|
+stg0501/stg0501_hrd.dat|Air Fleet (Expert)|ChaosControlStop Trigger x2 for blocked door / routes on Expert|
+stg0502/stg0502_hrd.dat|Iron Jungle (Expert)|ChaosControlStop Trigger at RailSplit / Dark upper route|
+stg0504/stg0504_cmn.dat|Lost Impact (All)|9 Extra Objects, 7 are SolidCollision Triggers, 1 Extra GUN Beetle and GUN Soldier at Checkpoint 8|
+stg0600/stg0600_cmn.dat|GUN Fortress (All)|Unknown|
+stg0600/stg0600_hrd.dat|GUN Fortress (Expert)|Two ChaosControlStop Triggers on upward Hero route areas|
+stg0602/stg0602_hrd.dat|Lava Shelter (Expert)|ChaosControlStop Trigger on Hero route|
+stg0604/stg0604_ds1.dat|Final Haunt (Destructibles)|Unknown. Note: Both have a Black Oak (Giant) which is abnormal for .ds1 files|
+stg0604/stg0604_hrd.dat|Final Haunt (Expert)|4x Duplicate Black Oak (x2 on Oak # 1, x2 on Oak # 3), x3 Platform Duplicate, x3 ChaosControlStop Triggers at Dark Route Flying Sections|
```

Comment:
I would prefer to have the order of this table to be: Stage & Type, Change, Layout File.  This will make things a bit more readable for runners that don't know or care to know about the technical files, but still found their way to this page to learn about the differences between the different builds of the game.

Reply:
Done